src/guide/extras/web-components.md

```diff
@@ -75,6 +75,30 @@ However, there could be rare cases where the data must be passed as a DOM proper
 <my-element .user="{ name: 'jack' }"></my-element>
 ```
 
+### Handling Events with Capital Letters
+
+The `v-on` directive handles lowercase and kebab-case events dispatched from custom elements. However, the `v-on` directive does not support listening for events whose names include capital letters. For example, adding `v-on:myInput` to a custom element adds a listener for events with the name `my-input`, which does not handle an event with the name `myInput`.
+
+If your app uses custom elements that dispatch events with capital letters, you can use a custom directive to add the correct event listener to the element.
+
+#### Example Custom Directive to Handle Specific Events
```

Comment:
I'm inclined to go straight from the previous paragraph into the example, without the extra heading.

I think we can also shorten **If your app uses** to just **If you use**

```suggestion
If you use custom elements that dispatch events with capital letters, you can use a custom directive to add the correct event listener to the element:
```